rediscluster/pipeline.py

```diff
@@ -142,144 +146,70 @@ def send_cluster_commands(self, stack, raise_on_error=True, allow_redirections=T
         `allow_redirections` If the pipeline should follow `ASK` & `MOVED` responses
         automatically. If set to false it will raise RedisClusterException.
         """
-        if self.refresh_table_asap:
```

Comment:
`StrictRedisCluster.execute_command` expects `refresh_table_asap` to be defined and blows up if it isn't.